lib/fog/bin/google.rb

```diff
@@ -24,25 +24,25 @@ def class_for(key)
 
     def [](service)
       @@connections ||= Hash.new do |hash, key|
-        hash[key] = case key
```

Comment:
I think the Ruby LSP on VSCode reformatted this. I don't see an issue here, but I can revert this if you prefer to avoid any unnecessary changes.